projects/uitdatabank/docs/entry-api/events/status.md

```diff
@@ -1,8 +1,141 @@
+* [Updating an event](./update.md) to postpone or cancel an event after creating it earlier
+* [Calendar info](../shared/calendar-info.md) for more info about the various calendar types and their properties
+
+## Permissions
+
+Anyone who [creates a new event](./create.md) can mark it as cancelled or postponed. For [updates to an existing event](./update.md), the usual permission checks apply.
+
+## The status property
+
+Every event automatically gets a `status` property after it is created.
+
+By default, it looks like this:
+
+```json
+{
+  "status": {
+    "type": "Available"
+  }
+}
+```
+
+The nested `type` property can either be `Available` (happening as scheduled), `Unavailable` (cancelled) or `TemporarilyUnavailable` (postponed). 
+
+The possible `type` values are intentionally worded in a generic way so that they match those of [the `status` property on places](../places/status.md), which makes it easier to query both events and places at the same time on their status property [using Search API](../../search-api/introduction.md).
+
+When the event has calendarType `single` or `multiple`, the objects inside its `subEvent` property will also automatically get the same `status` property. 
+
+For example on an event with multiple dates:
+
+```json
+{
+  "status": {
+    "type": "Available"
+  },
+  "calendarType": "multiple",
+  "subEvent": [
+    {
+      "startDate": "2023-01-18T17:30:00+01:00",
+      "endDate": "2023-01-18T23:00:00+01:00",
+      "status": {
+        "type": "Available"
+      }
+    },
+    {
+      "startDate": "2023-01-19T17:30:00+01:00",
+      "endDate": "2023-01-19T23:00:00+01:00",
+      "status": {
+        "type": "Available"
+      }
+    }
+  ]
+}
+```
+
+## Cancelling an event
+
+### With calendarType "single" or "multiple"
+
+To cancel an event with calendarType `single` or `multiple`, update the `status.type` property of the `subEvent` objects that should be cancelled and set it to `Unavailable`.
+
+Optionally, you can include a `reason` property to explain why the event is cancelled on that date. The `reason` property must be an object with language codes like `name`.
+
+For example to cancel one date of multiple:
+
+```json
+{
+  "calendarType": "multiple",
+  "subEvent": [
+    {
+      "startDate": "2023-01-18T17:30:00+01:00",
+      "endDate": "2023-01-18T23:00:00+01:00",
+      "status": {
+        "type": "Available"
+      }
+    },
+    {
+      "startDate": "2023-01-19T17:30:00+01:00",
+      "endDate": "2023-01-19T23:00:00+01:00",
+      "status": {
+        "type": "Unavailable",
+        "reason": {
+          "nl": "Afgelast wegens COVID-19 maatregelen."
+        }
+      }
+    }
+  ]
+}
+```
+
+Note that you may omit the `status` property on the top level, as it will automatically be set based on the `status` of each `subEvent`. If you include it anyway, it will be ignored.
+
+The top-level `status` is determined as follows:
```

Comment:
The logic for the derived `status` (3 possible statuses) is a bit more complex than the one for `bookingAvailability` (which has 2 possible statuses). See suggestion below.